### DISCUSSION TITLE
[Bug] Syncing is extremely slow

### DISCUSSION THREAD

**Discussion by luotinen** [Chia-Network/chia-blockchain#9862] [Support]
### What happened?

My wallet got stuck and stopped syncing. After trying multiple methods, I did a clean reinstall of Chia. 

Now the blockchain has been syncing for a week and it has reached 615,680/1,415,188. This implies a sync speed of 16 days (!) My CPU is 3950X and the internet speed is 400/40 (Mbps).

The long initial sync may discourage people from joining the project and encourage people to leave the project if they need to resync for any reason.

Is there a way to sync faster? Why does Chia sync so slowly?

Steps to replicate:
1. Remove Chia
2. Reinstall Chia
3. Attempt to sync

Expected behaviour:
Chia syncs in a matter of days not weeks

### Version

1.2.11

### What platform are you using?

Windows

### What ui mode are you using?

GUI

**Comment by emlowe** [comment]
You should be able to sync in 2 days or less with that h/w. Syncing speed is very dependant on the hard disk used. A very slow disk will result in very slow times.
Make sure to use an SSD to hold the DB.
The following is a interesting video about syncing from JM : https://www.youtube.com/watch?v=b59PBzCOet0

**Comment by luotinen** [comment]
I am now at 650,304/1,415,188. Since yesterday, in 19 hours, it has synced 34 688. The remaining sync time 775 hours or 32 days.

I will delete Chia once more, reinstall to a different drive (faster SSD) and see if it makes a difference.

**Reply by max918** [reply]
you can download the blockchain database file from some website directly and put it into the chia db folder, but it is not official recommend

https://www.chia-database.com/

**Comment by mattwhitfield** [comment]
I had to do this recently. 330/50 connection, took over 4 days to sync. DB is on a Samsung 960 Pro that shows good health in Magician.

Now, a few days later, my node has lost sync again and is currently about 4600 behind.

I am only farming 20gb, and I am considering ditching chia now this has happened a second time. Also the network has lost around 4EiB over the last few months. I don't really get why you would need a local dB of the entire blockchain to farm - last few blocks should be sufficient.

**Comment by luotinen** [comment]
I have now been syncing to SSD with 5950X two days and I am at 898,444/1,445,375 implying that currently (January 2022) the sync speed is:
32 days to HDD
4 days to SSD

At some point, solutions will be needed to both the large blockchain size and to the slow sync speed.

**Reply by mariano54** [reply]
Yes we are working on solutions to this problem

**Reply by Blackcode** [reply]
still working on? Slow as hell... 100.000 in one day

**Comment by luotinen** [comment]
The solution was to install Chia on a SSD drive. The ticket may now be closed.

The problem of (too) large ledger and slow sync speeds will remain, but they are being worked on.

**Comment by HVNC1** [comment]
можно просто скачать актуальную базу данных

**Comment by mariano54** [comment]
Some significant updates coming in version 1.3 (already in main)

**Comment by mattventura** [comment]
I see this too, even on 1.4.1-dev0. DB got corrupted again, so I had to revert to a snapshot from January, and now it has only synced about 100k in a day. The storage is fine in terms of performance (it's a ZFS pool with a ZIL), but I have to question what is going on in terms of disk access. Is it blocking downloads of new blocks until disk writes are complete or something like that? I almost never see a queue size of more than 2, and I/O request sizes are pretty small, so I think it's just not making good use of storage bandwidth (assuming that's the real culprit and not something network-related).
